notebooks/MAST/JWST/JWST_EngineeringDatabase_Retreival/JWST_EDB_Retrieval.ipynb

```diff
@@ -0,0 +1,560 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# JWST Engineering Data Retrieval"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial will show you how to retrieve JWST engineeering data and use it in the context of a python session. The [Engineering Data](https://outerspace.stsci.edu/pages/viewpage.action?spaceKey=DraftMASTDOCS&title=.Engineering+Data+v1.3) chapter of the [JWST Archive Manual](https://outerspace.stsci.edu/pages/viewpage.action?spaceKey=DraftMASTDOCS&title=.JWST+Archive+Manual+v1.2) describes how data from thousands of engineering telemetry points on JWST are stored in the Engineering Database in the form of timeseries. These data may be searched by means of an identifier, or **mnenomic**. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Some quantities of interest require more than one mnemonic (a *tuple*) for meaningful analysis. This tutorial illustrates how to retrieve a tuple of mnenomics and visualize the result. In the following example timeseries will be retrieved for mnemonics: `SA_ZADUCMDX` and `SA_ZADUCMDY` which are the *x-* and *y-* positions of the JWST [Fine Steering Mirror](https://jwst-docs.stsci.edu/jwst-observatory-hardware/jwst-telescope). \n",
+    "\n",
```

Comment:
Can we add a link to the authoritative listing of all mnenomics and their meanings here?